src/features/export-package-resources-modal/constants.js

```diff
@@ -130,6 +130,10 @@ const RESOURCE_EXPORT_FIELDS = [
 ];
 
 const PACKAGE_EXPORT_FIELDS = [
+  {
```

Comment:
Please also add to `RESOURCE_EXPORT_FIELDS` constants to export this from Resources too